### PR TITLE
v85.4.0

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# 85.4.0
+
+This minor release includes changes to the MDCTabBarView Beta component.
+
+## Component changes
+
```
Comment (codeman7) [inline CHANGELOG.md:6]: Please add the new APIs

Comment (romoore) [reply CHANGELOG.md:6]: Done. Please take a look.

**Review by codeman7 — CHANGES_REQUESTED** [review]
I don't see any version bumps

**Comment by romoore** [comment]
@codeman7 Good catch! I need to get sleep this weekend...

**Review by codeman7 — APPROVED** [review]

**Pull request merged**